lib/SQL/Translator/Parser/DBI/PostgreSQL.pm

```diff
@@ -134,6 +134,61 @@ ORDER BY 1;
         /
   ) or die "Can't prepare: $@";
 
+  my $trigger_select = $dbh->prepare(<<SQL) or die "Can't prepare trigger query: $@";
```

Comment:
I would like to see the description what is the result of these two blocks of code: 'trigger_select' and 'procedure_select'. Eg. `this block of code selects triggers and procedures from XXX which we can access later programmatically using YYY`